### DISCUSSION TITLE
Change the settings of a console at once

### DISCUSSION THREAD

**Discussion by FredBLRD** [webrcade/webrcade#128] [Ideas]
Hello, would it be possible to add an option to be able to completely modify the settings of a console and not to have to change the settings of a console on each game one by one independently?

An example on the Game Boy, change the hardware type : Super Game Boy 2

To change this setting, I have to do it on each game, it's very long when you have +300 games...

**Comment by raz0red** [comment]
Yeah, this is definitely something that could be added. Can you please submit an issue?

Thanks!
